Fixes #15644 - Add the ability to configure HSTS in NetBox (#15683)
netbox-community/netbox: docs/configuration/security.md

---

## SECURE_HSTS_INCLUDE_SUBDOMAINS

Default: False

If true, the `includeSubDomains` directive will be included in the HTTP Strict Transport Security (HSTS) header. This directive instructs the browser to apply the HSTS policy to all subdomains of the current domain.

---

## SECURE_HSTS_PRELOAD

Default: False

If true, the `preload` directive will be included in the HTTP Strict Transport Security (HSTS) header. This directive instructs the browser to preload the site in HTTPS. Browsers that use the HSTS preload list will force the site to be accessed via HTTPS even if the user types HTTP in the address bar.

---

## SECURE_HSTS_SECONDS

Default: 0

> If set to a non-zero integer value, the SecurityMiddleware sets the HTTP Strict Transport Security header on all responses that do not already have it.
> If the website must be accessed via HTTPS, it is recommended to set SECURE_HSTS_SECONDS. It will make the browser remember that the website must be accessed via HTTPS, blocking any HTTP request.  

Reviewer: ```suggestion
If set to a non-zero integer value, the SecurityMiddleware sets the HTTP Strict Transport Security (HSTS) header on all responses that do not already have it. This will instruct the browser that the website must be accessed via HTTPS, blocking any HTTP request.
```
